javascripts/discourse/services/enforce-edit-reason.js

```diff
@@ -40,10 +40,9 @@ export default class EnforceEditReason extends Service {
 
     const post = this.composer.model.post;
     const now = Date.now();
-    const gracePeriod = settings.edit_reason_grace_period * 1000;
+    const gracePeriod = settings.edit_reason_grace_period * 60 * 1000;
```

Comment:
Part 1: setting is in minutes, need to convert to milliseconds.

Reply:
Minor, but you could call it `gracePeriodMs`

Reply:
Thanks, will leave as is, I think, `now` is in `ms` as well, without a suffix.